app/tests/test_browse.py

```diff
@@ -30,715 +26,1594 @@ def test_standard_user_redirected_to_browse_transferring_body_when_accessing_bro
     )
 
 
-def test_browse_get(client: FlaskClient, mock_superuser):
-    """
-    Given a superuser accessing the browse page
-    When they make a GET request
-    Then they should see the browse page content.
-    """
-    mock_superuser(client)
-
-    response = client.get("/browse")
-
-    assert response.status_code == 200
-    assert b"Search for digital records" in response.data
-    assert b"You are viewing" in response.data
-    assert b"Everything available to you" in response.data
-
-
-def test_browse_submit_search_query(client: FlaskClient, mock_superuser):
-    """
-    Given a superuser accessing the browse page
-    When they make a POST request
-    Then they should see results in content.
-    """
-    mock_superuser(client)
-    create_multiple_test_records()
-
-    query = "test"
-    response = client.post("/browse", data={"query": query})
-
-    assert response.status_code == 200
-    assert b"Search for digital records" in response.data
-    assert b"Records found 11" in response.data
+class TestBrowse:
```

Comment:
can we move this test class / all browse all tests to their own test file since this file is really big

Reply:
I separated them with different classes for now, will pick up the split test cases in different files when splitting the browse routes

Reply:
ok